hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/TaskletExecutionService.java

```diff
@@ -378,17 +378,17 @@ private void runTasklet(TaskletTracker t) {
                 dismissTasklet(t);
             }
 
-            if (finestLogEnabled) {
+            if (fineLogEnabled) {
                 long elapsedMs = NANOSECONDS.toMillis((System.nanoTime() - start));
                 if (elapsedMs > COOPERATIVE_LOGGING_THRESHOLD) {
-                    logger.finest("Cooperative tasklet call of '" + t.tasklet + "' took more than "
+                    logger.fine("Cooperative tasklet call of '" + t.tasklet + "' took more than "
```

Comment:
I wonder if we won't have too many of these during soak tests, it's just 5ms. It also does `nanoTime` twice for each tasklet call, that's quite a lot and not negligible in benchmarks if FINE level is enabled.

Reply:
@viliam-durina We run soak tests with debug level. Even on debug level it produces dozens of GB logs.

Reply:
I'd rather keep the level as it was. I've never used this metric actually...